src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -78,6 +78,8 @@ public SysFsDriver()
         /// <param name="pinNumber">The pin number in the driver's logical numbering scheme.</param>
         protected internal override void OpenPin(int pinNumber)
         {
+            const int retryDelayMaximumMillis = 1000; // wait 1 second max for udev to set the permissions
+            Diagnostics.Stopwatch retryStopwatch;
```

Comment:
this should be declared as late as possible

Reply:
Done in latest PR update